toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+        <translation>Flexibel</translation>
+    </message>
+    <message>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <source>Vertex Name:</source>
+        <translation>Vertex-Name</translation>
+    </message>
+    <message>
+        <source>Allow Stretching</source>
+        <translation>Strecken erlauben</translation>
+    </message>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+    <message>
+        <source>Keep Distance</source>
+        <translation>Abstand beibehalten</translation>
+    </message>
+    <message>
+        <source>Global Key</source>
+        <translation>Globale Taste</translation>
+    </message>
+    <message>
+        <source>A group of skeletons already exists for current column. Replacing it will also substitute any existing vertex animation.
+
+Do you want to continue?</source>
+        <translation>Eine Gruppe Skelette existiert bereits für die aktuelle Spalte. Sie zu ersetzen, wird jegliche vorhandene Vertex-Animationen austauschen.</translation>
+    </message>
+    <message>
+        <source>Ok</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Abbruch</translation>
+    </message>
+    <message>
+        <source>Copy Skeleton</source>
+        <translation>Skelett kopieren</translation>
+    </message>
+    <message>
+        <source>Paste Skeleton</source>
+        <translation>Skelett einfügen</translation>
+    </message>
+    <message>
+        <source>Delete Vertex</source>
+        <translation>Vertex löschen</translation>
+    </message>
+    <message>
+        <source>Set Key</source>
+        <translation>Taste setzen</translation>
+    </message>
+    <message>
+        <source>Set Rest Key</source>
+        <translation>Rast-Taste setzen</translation>
+    </message>
+    <message>
+        <source>Set Global Key</source>
+        <translation>Globale Taste setzen</translation>
+    </message>
+    <message>
+        <source>Set Global Rest Key</source>
+        <translation>Globale Rast-Taste setzen</translation>
+    </message>
+    <message>
+        <source>Show Mesh</source>
+        <translation>Mesh zeigen</translation>
+    </message>
+    <message>
+        <source>Show Rigidity</source>
+        <translation>Steifheit zeigen</translation>
+    </message>
+    <message>
+        <source>Show SO</source>
+        <translation>SO zeigen</translation>
+    </message>
+    <message>
+        <source>Show Skeleton Onion Skin</source>
+        <translation>Skelett Onion Skin zeigen</translation>
+    </message>
+    <message>
+        <source>The previous vertex name will be discarded, and all associated keys will be lost.
+
+Do you want to proceed?</source>
+        <translation>Der vorherige Vertex-Name wird verworfen, und all assoziierten Tasten gehen verloren.</translation>
```

Comment:
"all assoziierten Tasten" should probably be "alle zugewiesenen Tasten".

Also missing "Möchten Sie forfahren?".